centreon/tests/e2e/features/Api-Token/02-api-token-listing.feature

```diff
@@ -0,0 +1,21 @@
+@ignore
+@REQ_MON24867
+Feature: List Existing API Tokens in Administration
+
+  As an administrator
+  I want to view all existing API tokens under "Administration > App token"
+  So that I can manage and oversee token details
+
+  Background:
+    Given I am logged in as an administrator
+    And I am on the "Administration" section
+
+  Scenario: View Existing API Tokens
```

Comment:
is this a scenario with example?
if yes, it should be "Scenario Outline", and parameters should be delimited